www/js/banidb/index.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
       }
       case CONSTS.SEARCH_TYPES.ANG: // Ang
         searchCol = 'PageNo';
+        howManyRows = 1000;
```

Comment:
is this the max for any ang?

Reply:
It's just ridiculously high to make sure all cases are covered.